bon/tests/integration/builder/attr_getter.rs

```diff
@@ -0,0 +1,60 @@
+use crate::prelude::*;
+
+#[test]
+fn test_struct() {
+    #[derive(Debug, Builder)]
+    #[builder(derive(Debug, Clone))]
+    #[allow(dead_code)]
+
+    struct Sut<T> {
+        #[builder(getter, start_fn)]
+        x1: u32,
```

Comment:
Let's add calls to `get_x1`, `x2_with_custom_name`, `get_x3`, `get_generic_option_getter` in the test.

In fact, you'll notice that if you set `maybe_generic_option_getter(None)`, and then call `get_generic_option_getter`, you'll get undefined behavior.